.github/workflows/build-android-app.yml

```diff
@@ -25,7 +25,7 @@ jobs:
       with:
         node-version: 16
         cache: 'npm'
-
+    - run: npm i -g @mapbox/node-pre-gyp
     - run: npm i
```

Comment:
I think this can be removed.  I mentioned this at some point, but a lot of older webrtc npm packages (simple-peer, etc.) use a package called `wrtc`, which will not build without this `node-pre-gyp` thing installed in the system.  I'm pretty sure I've removed anything that referenced that package, though.

Reply:
Confirmed.  deleted those changes and the build works fine.